gitlab/mixins.py

```diff
@@ -56,7 +56,7 @@ class HeadMixin(_RestManagerBase):
     @exc.on_http_error(exc.GitlabHeadError)
     def head(
         self, id: Optional[Union[str, int]] = None, **kwargs: Any
-    ) -> "requests.structures.CaseInsensitiveDict[Any]":
+    ) -> Union["requests.structures.CaseInsensitiveDict[Any]", Any]:
```

Comment:
Why is the `Any` needed? And for all the other ones.

Reply:
Because the httpx client function will return different type which I didn't want to set at this stage. Right now it is a placeholder for the next implementation phase.